client/src/pages/Callback.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {useEffect} from "react";
+import axios from "axios";
+import GlobalLayout from "../components/common/GlobalLayout";
+import Spinner from "../components/common/Spinner";
+
+interface props {
+  location: string,
+  history: string[]
+}
+
+const Callback: React.FC<props> = ({location, history}: props) => {
+  const githubBaseUrl = "/api/signin/github";
+
+  useEffect(() => {
+    const getToken = async () => {
+      const code: string = location.search.toString().split("=")[1];
```

Comment:
```suggestion
      const code: string = new URLSearchParams(location.search).get("code");
```

[URLSearchParams](https://developer.mozilla.org/ko/docs/Web/API/URLSearchParams) 라는 것도 있어요 :)